src/nectarchain/trr_test_suite/utils.py

```diff
@@ -190,6 +193,89 @@
 
 adc_to_pe = 58.0
 
+
+def get_bad_pixels_list():
+    # List of modules and pixels to be rejected
+    df = pd.read_json("resources/bad_pix_module.json")
+    modules_list = np.array(df.bad_module[0])
+    pix_list = np.array(df.bad_pixel[0])
+
+    pix_nos = np.arange(7)
+
+    print("module_to_pix ", modules_list)
+    module_to_pix = modules_list[:, None] * 7 + pix_nos
+    combined = np.concatenate([pix_list, module_to_pix.ravel()])
+
+    bad_pix_list = np.unique(combined)
+
+    return bad_pix_list
+
+
+def get_adc_to_pe(temperature):
+    temp = np.array([-10, -5, 0, 5, 10, 14, 20, 25])
+    runs = np.array([6853, 6775, 6718, 6589, 7191, 7000, 7123, 7066])
+
+    if temperature in temp:
+        run_no = runs[temp == temperature][0]
+        gain_file_name = (
+            "resources/gain_spe/gain_spe/"
```

Comment:
This file is not included in the `resources` directory. Should it be added ? Or should this input file passed as argument ?